Dockerfile

```diff
@@ -50,6 +50,13 @@ RUN apt-get update \
     python3-enchant \
   && locale-gen en_US.UTF-8
 
+ARG USER_UID=1000
+ARG USER_GID=$USER_UID
+ARG OSTYPE=Linux
+
+RUN bash -c "[ \"$OSTYPE\" = \"Linux\" ] && groupadd --gid \"${USER_GID}\" dependabot" \
```

Comment:
Question: why do we only want to create the group if `OSTYPE = Linux`?

When `OSTYPE != Linux`, how does the `chown` on L59 work? We haven't created the `dependabot` group...

Reply:
This was my attempt at fixing an issue on mac where the `USER_GID` already existed so the `groupadd` failed, but you're right, the chown should fail as the group is surely called something other than `dependabot`.

Could we do something like `chown ${USER_UID}:${USER_GID}` instead? There might also be a better way to deal with this, not sure why I've never run into it before with the `docker-dev-shell`

Reply:
> Could we do something like chown ${USER_UID}:${USER_GID} instead?

SGTM.

Alternatively, do we need to align `USER_GID` with the host? As long as `USER_UID` matches the permissions will work out...